pkg/config/stepmeta.go

```diff
@@ -80,15 +80,22 @@ type StepSecrets struct {
 // Container defines an execution container
 type Container struct {
 	//ToDo: check dockerOptions, dockerVolumeBind, containerPortMappings, sidecarOptions, sidecarVolumeBind
-	Command         []string    `json:"command"`
-	EnvVars         []EnvVar    `json:"env"`
-	Image           string      `json:"image"`
-	ImagePullPolicy string      `json:"imagePullPolicy"`
-	Name            string      `json:"name"`
-	ReadyCommand    string      `json:"readyCommand"`
-	Shell           string      `json:"shell"`
-	WorkingDir      string      `json:"workingDir"`
-	Conditions      []Condition `json:"conditions,omitempty"`
+	Command         []string     `json:"command"`
+	EnvVars         []EnvVar     `json:"env"`
+	Image           string       `json:"image"`
+	ImagePullPolicy string       `json:"imagePullPolicy"`
+	Name            string       `json:"name"`
+	ReadyCommand    string       `json:"readyCommand"`
+	Shell           string       `json:"shell"`
+	WorkingDir      string       `json:"workingDir"`
+	Conditions      []Condition  `json:"conditions,omitempty"`
+	VolumeBind      []VolumeBind `json:"volumeBind,omitempty"`
+}
+
+// VolumeBind defines an environment variable
+type VolumeBind struct {
+	Name  string `json:"name"`
```

Comment:
Does Name-Value really fit here? Isn't it rather something like host path and container path?

Reply:
And it has `mountPath` and `name` as properties.